.goreleaser.yml

```diff
@@ -72,31 +72,23 @@ nfpms:
         file_info:
           mode: 0700
       - src: build/package/config.yaml
-        dst: /etc/newrelic-agent-control/config.yaml
+        dst: /etc/newrelic-agent-control/local-data/agent-control/config.yaml
         type: config
         file_info:
           mode: 0600
       - src: build/package/newrelic-agent-control.service
         dst: /lib/systemd/system/newrelic-agent-control.service
       - src: build/package/newrelic-agent-control.conf
-        dst: /etc/newrelic-agent-control/newrelic-agent-control.conf
+        dst: /etc/newrelic-agent-control/systemd-env.conf
```

Comment:
I was not aware of this change, have you chosen the name together? 

If so I am fine with it, otherwise I would like revisit it  😄